client/src/pages/UsersPage.js

```diff
@@ -16,19 +16,21 @@ UsersPage.propTypes = {
   ),
 }
 
-export default function UsersPage({ users, title }) {
+export default function UsersPage({ users, title, deleteUser }) {
   return (
     <Wrapper>
       <Header>{title}</Header>
       <ListWrapper>
-        {users.map(({ id, skills, name, phone, email, department }) => (
-          <SingleEntry key={id}>
+        {users.map(({ _id, skills, name, phone, email, department }) => (
```

Comment:
I think you could hand over user as a single parameter instead of _id, skills, name etc.
`users.map((user) => ... <User user={user} />`
instead of
`users.map((id, skills, name etc) => ... <User id=, skills=, name= etc>`